### PR TITLE
prefab/curl-ssl minor documentation update

### PR DESCRIPTION
**Pull request by ggfan**
Update READM.md in prefab/curl-ssl sample with more Prefab instructions, no functional changes.

### CODE REVIEW

File: prefab/curl-ssl/README.md [hunk 1]
```diff
@@ -1,12 +1,27 @@
 # curl-ssl
 
-The curl-ssl sample shows how to import native dependencies from Maven. This
-sample uses [curl], [OpenSSL], and [JsonCpp] to display a list of the most
-recent 10 reviews submitted to AOSP's code review system.
+The curl-ssl sample shows how to use [Prefab] to import native dependencies from [Google Maven]. This
```
Comment (DanAlbert) [inline prefab/curl-ssl/README.md:3]: I'd revert the change to this line. Prefab is an implementation detail that's not important, and gmaven is just one of the repositories this works with,

File: prefab/curl-ssl/README.md [hunk 2]
```diff
@@ -1,12 +1,27 @@
 # curl-ssl
 
-The curl-ssl sample shows how to import native dependencies from Maven. This
-sample uses [curl], [OpenSSL], and [JsonCpp] to display a list of the most
-recent 10 reviews submitted to AOSP's code review system.
+The curl-ssl sample shows how to use [Prefab] to import native dependencies from [Google Maven]. This  
+sample uses [curl], [OpenSSL], and [JsonCpp] to display a list of the most  
+recent 10 reviews submitted to AOSP's code review system.  
+To use [Prefab] with Android Studio 4.0+, simply follow the 3 steps:
```
Comment (DanAlbert) [inline prefab/curl-ssl/README.md:6]: Android Gradle Plugin, not Android Studio. Android Studio is not relevant to this sample.

Also, need an empty line to start a new paragraph.

File: prefab/curl-ssl/README.md [hunk 3]
```diff
@@ -1,12 +1,27 @@
...
 
-[curl]: https://curl.haxx.se/
-[OpenSSL]: https://www.openssl.org/
+* turn on prefab feature in [gradle.properties] 
+* declare the dependencies in app's [build.gradle]
```
Comment (DanAlbert) [inline prefab/curl-ssl/README.md:9]: s/app's/the app module's/, maybe?

File: prefab/curl-ssl/README.md [hunk 4]
```diff
@@ -1,12 +1,27 @@
...
+* add the dependencies into native module's [android.mk] or [CMakeLists.txt]
```
Comment (DanAlbert) [inline prefab/curl-ssl/README.md:10]: into the native module's

(also, capitalize the first letter of each of these)

File: prefab/curl-ssl/gradle.properties [hunk 5]
```diff
@@ -19,4 +19,10 @@ android.useAndroidX=true
 android.enableJetifier=true
 # Kotlin code style for this project: "official" or "obsolete":
 kotlin.code.style=official
+
+# enable Prefab feature in gradle plugin 4.0+
 android.enablePrefab=true
+
+# optional: pick up the latest Prefab release from https://github.com/google/prefab/releases
+android.prefabVersion=1.0.0
```
Comment (DanAlbert) [inline prefab/curl-ssl/gradle.properties:27]: This isn't necessary, just remove it.

File: prefab/curl-ssl/gradle.properties [hunk 6]
```diff
@@ -19,4 +19,10 @@ android.useAndroidX=true
 android.enableJetifier=true
 # Kotlin code style for this project: "official" or "obsolete":
 kotlin.code.style=official
+
+# enable Prefab feature in gradle plugin 4.0+
```
Comment (DanAlbert) [inline prefab/curl-ssl/gradle.properties:23]: This is actually 4.0 only. 4.1 is moving things from gradle.properties into build.gradle, so this won't work in 4.1.

I'd change this to "Enable importing prefab depenencies".

File: prefab/curl-ssl/README.md (hunk 2 above)
Comment (ggfan) [reply prefab/curl-ssl/README.md]: done

File: prefab/curl-ssl/README.md (hunk 1 above)
Comment (ggfan) [reply prefab/curl-ssl/README.md]: done

File: prefab/curl-ssl/README.md (hunk 3 above)
Comment (ggfan) [reply prefab/curl-ssl/README.md]: done

File: prefab/curl-ssl/README.md (hunk 4 above)
Comment (ggfan) [reply prefab/curl-ssl/README.md]: done

File: prefab/curl-ssl/gradle.properties (hunk 6 above)
Comment (ggfan) [reply prefab/curl-ssl/gradle.properties]: done! this is good one: when 4.1 coming along, this would be updated accordingly

File: prefab/curl-ssl/gradle.properties (hunk 5 above)
Comment (ggfan) [reply prefab/curl-ssl/gradle.properties]: done.

**Comment by ggfan** [comment]
@DanAlbert :addressed the review suggestions, ready to submit,  please take one more look, thanks!

File: prefab/curl-ssl/README.md [hunk 7]
```diff
@@ -4,9 +4,24 @@ The curl-ssl sample shows how to import native dependencies from Maven. This
 sample uses [curl], [OpenSSL], and [JsonCpp] to display a list of the most
 recent 10 reviews submitted to AOSP's code review system.
 
-[curl]: https://curl.haxx.se/
-[OpenSSL]: https://www.openssl.org/
+To use [Prefab] with Android Gradle Plugin 4.0+, simply follow the following 3 steps:
+
+* Turn on prefab feature in [gradle.properties]
+* Declare the dependencies in the app's module [build.gradle]
```
Comment (DanAlbert) [inline prefab/curl-ssl/README.md:10]: app module's, not app's module

File: prefab/curl-ssl/README.md [hunk 8]
```diff
@@ -4,9 +4,24 @@ The curl-ssl sample shows how to import native dependencies from Maven. This
...
+* Add the dependencies into the native module's [android.mk] or [CMakeLists.txt]
+
+From here on, the headers and native libs in the dependencies are [ready to use]!
+
+[Prefab]:https://github.com/google/prefab
```
Comment (DanAlbert) [inline prefab/curl-ssl/README.md:15]: Use https://google.github.io/prefab instead. It's a better landing page and it does link to the repo.

File: prefab/curl-ssl/gradle.properties [hunk 9]
```diff
@@ -19,4 +19,8 @@ android.useAndroidX=true
 android.enableJetifier=true
 # Kotlin code style for this project: "official" or "obsolete":
 kotlin.code.style=official
+
+# Enable importing Prefab dependencies
 android.enablePrefab=true
+
```
Comment (DanAlbert) [inline prefab/curl-ssl/gradle.properties:25]: Looks like you accidentally added a few empty lines to the end.

File: prefab/curl-ssl/README.md (hunk 7 above)
Comment (ggfan) [reply prefab/curl-ssl/README.md]: done

File: prefab/curl-ssl/README.md (hunk 8 above)
Comment (ggfan) [reply prefab/curl-ssl/README.md]: it is better landing page, thanks!

File: prefab/curl-ssl/gradle.properties (hunk 9 above)
Comment (ggfan) [reply prefab/curl-ssl/gradle.properties]: that is true: removed!

**Comment by ggfan** [comment]
@DanAlbert: just force pushed(-f) a patch, addressed your additional reviews.  please take a look, thanks!

File: prefab/curl-ssl/README.md [hunk 10]
```diff
@@ -4,9 +4,24 @@ The curl-ssl sample shows how to import native dependencies from Maven. This
 sample uses [curl], [OpenSSL], and [JsonCpp] to display a list of the most
 recent 10 reviews submitted to AOSP's code review system.
 
-[curl]: https://curl.haxx.se/
-[OpenSSL]: https://www.openssl.org/
+To use [Prefab] with Android Gradle Plugin 4.0+, simply follow the following 3 steps:
```
Comment (irataxy) [inline prefab/curl-ssl/README.md:7]: ...follow these 3 steps:

File: prefab/curl-ssl/README.md [hunk 11]
```diff
@@ -4,9 +4,24 @@ The curl-ssl sample shows how to import native dependencies from Maven. This
...
+
+* Turn on prefab feature in [gradle.properties]
```
Comment (irataxy) [inline prefab/curl-ssl/README.md:9]: * Enable the prefab ...

**Comment by ggfan** [comment]
addressed the feedbacks, ready to merge now, thank you!
